Tests/StreamChatTestTools/TestData/PhotoMetaData.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+@testable import StreamChat
+
+public struct PhotoMetadata: Codable, Equatable {
```

Comment:
Why do we need this one?

Reply:
@martinmitrevski this tool is primarily used for testing image attachments payloads.

Reply:
thanks @bielikb, I was confused about the long/lat, but I guess that's also provided in the attachments metadata.